index.html

```diff
@@ -10,125 +10,143 @@
   <body>
     <header>
       <a id="logo">Farida's Logo</a>
+      <nav id="top">
+        <a>Portfolio</a>
+        <a>About</a>
+        <a>Contact</a>
+      </nav> 
       <nav>
         <img src="./images/Union.jpg" alt="menue bar" id="menue-button">
       </nav>
     </header>
     <main>
       <section id="intro">
-        <h1>I'm Farida Glad to see you!</h1>
-        <p id="bio">I’m a software developer! I can help you build a product, 
-          feature or website Look through some of my work and experience! 
-          If you like what you see and have a project you need coded, 
-          don’t hestiate to contact me.
-        </p>
-        
-        <h3>let's connect</h3>
-        <ul class="social">
-          <li><img src="./images/Vector.png" alt="twitter"></li>
-          <li><img src="./images/Linkedin-icon.jpg" alt="linkedin"></li>
-          <li><img src="./images/Vector.jpg" alt="email"></li>
-          <li><img src="./images/Vector-2.jpg" alt="github"></li>
-          <li><img src="./images/Vector-3.jpg" alt="angel-list"></li>
-        </ul>
+        <div id="intro-content">
+          <h1>I'm Farida</h1>
+          <h1>Glad to see you!</h1>
+          <p>I’m a software developer! I can help you build a product, 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+          </p>
+          
+          <h3>let's connect</h3>
+          <ul class="social">
+            <li><img src="./images/Vector.png" alt="twitter"></li>
+            <li><img src="./images/Linkedin-icon.jpg" alt="linkedin"></li>
+            <li><img src="./images/Vector.jpg" alt="email"></li>
+            <li><img src="./images/Vector-2.jpg" alt="github"></li>
+            <li><img src="./images/Vector-3.jpg" alt="angel-list"></li>
+          </ul>
+        </div>
       </section>
 
       <section class="previous-work">
         <section class="project">
```

Comment:
- [ ] You should use the padding and margin as in the Figma design for the project cards. The gap between the image and the project details is a bit wide on your implementation while the padding left and right are narrow unlike the Figma design. Please take a look at the following snapshots. 
#### Figma
![card-gap-pading-figma](https://user-images.githubusercontent.com/44057863/166105922-d855faf6-ac8e-4964-b6cd-112583918c72.png)

#### Implementatio
![card-padding](https://user-images.githubusercontent.com/44057863/166105942-17f16ce9-4c63-4cc2-bfb2-cac878427af4.png)